packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx

```diff
@@ -196,12 +196,25 @@ export default function TimeSliderQuickFilter({
     <div style={{ paddingTop: 10 }}>
       <H6>Timeline help</H6>
       <p>
+        Black bars indicate when <i>in time</i> there is data that...
         <ul>
-          <li>Black bars indicate when in time there is available data</li>
-          <li>Permanent filters are applied if applicable</li>
+          <li>is located anywhere on Earth*, not just currently in view</li>
           <li>
-            You currently have {filters?.length} active permanent filter(s)
+            has a value for <b>{variableMetadata?.variable.displayName}</b>
           </li>
+          <li>satisfies any filters you have applied</li>
+          <li>
+            has values for the variable currently displayed on markers,
+            including custom-configurations if applicable
+          </li>
+          <li style={{ listStyleType: 'none' }}>
```

Comment:
Ideally this would not be a list item, since it isn't semantically speaking.

Reply:
For screen-reader purposes?

Reply:
Yep, that's the main reason

Reply:
I wanted it to be the same indentation level but not have to eyeball the indentation level. But putting it back to the <p> level looks fine, especially with parentheses.

Reply:
I was about to say that I also don't like that it's indented with the rest of the list. To me, it gets lost. I expect "footnotes" to appear not indented.